### PR TITLE
Test #189

### PR DESCRIPTION
**Pull request by webflo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,14 @@ env:
...
-    - RELEASE=dev
+    - RELEASE=stable,COMPOSER_CHANNEL=stable
+    - RELEASE=dev,COMPOSER_CHANNEL=stable
+    - RELEASE=stable,COMPOSER_CHANNEL=snapshot
 
 before_install:
   - echo 'sendmail_path = /bin/true' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - phpenv config-rm xdebug.ini
-  - composer --verbose self-update
+  - composer --verbose self-update --$COMPOSER_CHANNEL
```
Comment (alcohol) [inline .travis.yml:21]: This syntax does not seem to work as expected.

```
// RELEASE=stable,COMPOSER_CHANNEL=snapshot
$ composer --verbose self-update --$COMPOSER_CHANNEL
Updating to version 1.2.0 (stable channel).
```

**Comment by webflo** [comment]
Test in drupal core fails because of https://github.com/symfony/symfony/pull/19321 https://github.com/symfony/http-foundation/commit/a52a882549efdd48e80816e3cf5edd25f6ac2fcd

**Comment by webflo** [comment]
Core fix in https://www.drupal.org/node/2770673 and https://www.drupal.org/node/2776367

**Comment by webflo** [comment]
@rodrigoaguilera Please review

**Comment by rodrigoaguilera** [comment]
Thanks!

I updated Scripthandler on my project with the latest changes and now it doesn't complain about the composer version.

**Comment by wimleers** [comment]
Related: https://github.com/symfony/symfony/issues/20562

**Pull request merged**